hoomd/box.py

```diff
@@ -455,12 +546,17 @@ def to_matrix(self):
         """(3, 3) `numpy.ndarray` `float`: The upper triangular matrix that \
         defines the box.
 
-        .. code-block:: python
+        .. code-block::
```

Comment:
In line 227 you removed `.. code-block::` completely. I would make it consistent throughout the whole file.

Reply:
I made this change intentionally. This block is not a code example, it is a monospace font formatted matrix. I removed the sphinx `.. code-block::` directive and used the restructured text literal block instead: https://docutils.sourceforge.io/0.4/docs/ref/rst/restructuredtext.html#literal-blocks

All visible code examples that execute in sybil must be preceded by the sphinx block `.. code-block:: python`: https://sybil.readthedocs.io/en/latest/rest.html#python - other `.. code-block::` blocks are ignored by sybil.

Reply:
I added a math formatted matrix.